widgetry/src/text.rs

```diff
@@ -271,6 +271,7 @@ impl Text {
         self.render(assets).get_dims()
     }
 
+    // TODO: Combine all render_foo methods to be render(into<Assets>)?
```

Comment:
WDYT about this proposed cleanup - it would just make widgetry a tiny bit easier to use.